### PR TITLE
New Ads design [only on test]

### PR DESCRIPTION
**Pull request by DarioR01**
Resolves [WSTEAM1-21](https://jira.dev.bbc.co.uk/browse/WSTEAM1-21)

**Overall change:**
~~All the logic for ads for the Optimo article seems to be already there, therefore I only applied the style changes.~~

Ads playground in test for UX

**Code changes:**

- ~~Set AdSection background colour to Grey-3  #E6E8EA in Optimo articles. (Currently unable to add the new colour on Psammead due to the soft change applied by the Psammead migration).~~
- ~~Remove white spacing between leading ad and navigation bar in both CPS and Optimo articles.~~
- Set AdSection background colour to Grey-3  #E6E8EA.
- Remove white spacing between leading ad and navigation bar.
- Refactor SlotStyle Paddings. 
- **These changes should only be deployed in test environment only!!!**
---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by amoore108 — DISMISSED** [review]

**Review by emilysaffron — DISMISSED** [review]

**Review by Louis-Matsika — DISMISSED** [review]

File: src/app/containers/Ad/Amp/index.jsx [hunk 1]
```diff
@@ -23,7 +23,8 @@ import { ampLeaderboardStyles, ampMpuStyles } from '../utilities/adSlotStyles';
 const AccessDiv = props => <div {...props} />;
 
 const AdSection = styled.section`
-  background-color: ${C_LUNAR_LIGHT};
+  background-color: ${({ pageType }) =>
+    pageType === 'article' ? '#E6E8EA' : C_LUNAR_LIGHT};
```
Comment (MuhammadAHussain) [inline src/app/containers/Ad/Amp/index.jsx]: Worth putting a comment to move this colour into a variable when the migration is complete?

**Review by MuhammadAHussain — DISMISSED** [review]
LGTM 👍

**Review by amoore108 — APPROVED** [review]
Looks good 👍🏼

File: src/app/containers/Ad/Canonical/index.jsx [hunk 2]
```diff
@@ -3,17 +3,20 @@ import { useLocation } from 'react-router-dom';
...
+import { C_LUNAR_LIGHT } from '#legacy/psammead-styles/src/colours';
 import getAdsAriaLabel from '../utilities/getAdsAriaLabel';
 import { leaderboardStyles, mpuStyles } from '../utilities/adSlotStyles';
 
+// move colour #E6E8EA in the legacy folder as C_GREY_3 once psammead migration is completed
+const C_GREY_3 = '#E6E8EA';
+
 const AdContainer = styled.section`
-  background-color: ${C_LUNAR_LIGHT};
+  background-color: ${() => (isLive() ? C_LUNAR_LIGHT : C_GREY_3)};
```
Comment (jroebu14) [inline src/app/containers/Ad/Canonical/index.jsx]:
```suggestion
  background-color: ${isLive() ? C_LUNAR_LIGHT : C_GREY_3};
```

you would only use a function here to get component props

File: src/app/containers/Ad/utilities/adSlotStyles.js [hunk 3]
```diff
@@ -114,7 +121,8 @@ export const mpuStyles = `
     visibility: visible;
   }
   @media (min-width: ${MPU_WIDTH_MIN_PADDING}) {
-    padding: 0 ${GEL_SPACING};
+    padding: ${() =>
+      isLive() ? `0 ${GEL_SPACING}` : `${GEL_SPACING_DBL} 0 ${GEL_SPACING} 0`};
```
Comment (jroebu14) [inline src/app/containers/Ad/utilities/adSlotStyles.js]:
```suggestion
    padding: ${isLive() ? `0 ${GEL_SPACING}` : `${GEL_SPACING_DBL} 0 ${GEL_SPACING} 0`};
```

File: src/app/containers/Ad/utilities/adSlotStyles.js [hunk 4]
```diff
@@ -65,25 +68,29 @@ export const leaderboardStyles = `
...
     min-height: ${LEADERBOARD_HEIGHTS.GROUP_3};
+    ${() => !isLive() && `padding: ${GEL_SPACING_QUAD} 0;`}
   }
   @media (min-width: ${LARGE_LEADERBOARD_WIDTH_MIN}) {
     min-height: ${LEADERBOARD_HEIGHTS.LARGE};
-    padding: ${GEL_SPACING} ${GEL_SPACING_DBL};
+    padding: ${() =>
+      isLive()
+        ? `${GEL_SPACING} ${GEL_SPACING_DBL};`
+        : `${GEL_SPACING_TRPL} 0`};
```
Comment (jroebu14) [inline src/app/containers/Ad/utilities/adSlotStyles.js]:
```suggestion
    padding: ${
      isLive()
        ? `${GEL_SPACING} ${GEL_SPACING_DBL};`
        : `${GEL_SPACING_TRPL} 0`};
```

File: src/app/containers/Ad/utilities/adSlotStyles.js [hunk 5]
```diff
@@ -65,25 +68,29 @@ export const leaderboardStyles = `
...
   @media (min-width: ${LEADERBOARD_WIDTH_MIN_PADDING}) {
-    padding: 0 ${GEL_SPACING};
+    padding: ${() => (isLive() ? `0 ${GEL_SPACING}` : `${GEL_SPACING_DBL} 0`)};
   }
   @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
     min-height: ${LEADERBOARD_HEIGHTS.GROUP_2};
   }
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
     min-height: ${LEADERBOARD_HEIGHTS.GROUP_3};
+    ${() => !isLive() && `padding: ${GEL_SPACING_QUAD} 0;`}
```
Comment (jroebu14) [inline src/app/containers/Ad/utilities/adSlotStyles.js]:
```suggestion
    ${!isLive() && `padding: ${GEL_SPACING_QUAD} 0;`}
```

File: src/app/containers/Ad/utilities/adSlotStyles.js [hunk 6]
```diff
@@ -65,25 +68,29 @@ export const leaderboardStyles = `
...
+    ${isLive() && `margin-top: ${GEL_SPACING}`};
     min-height: ${LEADERBOARD_HEIGHTS.GROUP_1};
     display: flex;
     flex-direction: column;
     justify-content: center;
     visibility: visible;
   }
   @media (min-width: ${LEADERBOARD_WIDTH_MIN_PADDING}) {
-    padding: 0 ${GEL_SPACING};
+    padding: ${() => (isLive() ? `0 ${GEL_SPACING}` : `${GEL_SPACING_DBL} 0`)};
```
Comment (jroebu14) [inline src/app/containers/Ad/utilities/adSlotStyles.js]:
```suggestion
    padding: ${isLive() ? `0 ${GEL_SPACING}` : `${GEL_SPACING_DBL} 0`};
```

File: src/app/containers/Ad/Canonical/index.jsx (hunk 2 above)
Comment (DarioR01) [reply src/app/containers/Ad/Canonical/index.jsx]: Fixed this, thanks for spotting this 👍

File: src/app/containers/Ad/Canonical/index.jsx [hunk 7]
```diff
@@ -3,17 +3,20 @@ import { useLocation } from 'react-router-dom';
...
 import pathOr from 'ramda/src/pathOr';
 import { ServiceContext } from '#contexts/ServiceContext';
 import { RequestContext } from '#contexts/RequestContext';
 import isLive from '#lib/utilities/isLive';
 import useOperaMiniDetection from '#hooks/useOperaMiniDetection';
+import { C_LUNAR_LIGHT } from '#legacy/psammead-styles/src/colours';
 import getAdsAriaLabel from '../utilities/getAdsAriaLabel';
 import { leaderboardStyles, mpuStyles } from '../utilities/adSlotStyles';
 
+// move colour #E6E8EA in the legacy folder as C_GREY_3 once psammead migration is completed
```
Comment (jroebu14) [inline src/app/containers/Ad/Canonical/index.jsx]: can we do this now?

**Review by jroebu14 — APPROVED** [review]
just one comment. Looks good 👍

File: src/app/containers/Ad/Canonical/index.jsx (hunk 7 above)
Comment (DarioR01) [reply src/app/containers/Ad/Canonical/index.jsx]: Yes we can.

**Pull request merged**